npm/npm.go

```diff
@@ -77,6 +80,58 @@ func (npMgr *NetworkPolicyManager) GetClusterState() telemetry.ClusterState {
 	return npMgr.clusterState
 }
 
+// SendAiMetrics :- send NPM metrics using AppInsights
+func (npMgr *NetworkPolicyManager) SendAiMetrics() {
+	var (
+		aiConfig = aitelemetry.AIConfig{
+			AppName:                   util.AzureNpmFlag,
+			AppVersion:                npMgr.version,
+			BatchSize:                 32768,
+			BatchInterval:             30,
+			RefreshTimeout:            15,
+			DebugMode:                 true,
+			GetEnvRetryCount:          5,
+			GetEnvRetryWaitTimeInSecs: 3,
+		}
+		th, err = aitelemetry.NewAITelemetry("", aiMetadata, aiConfig)
+		heartbeat = time.NewTicker(time.Minute * heartbeatIntervalInMinutes).C
+		customDimensions = map[string]string{"ClusterID": util.GetClusterID(npMgr.nodeName),
```

Comment:
why the assignment was in next line?

Reply:
what do you mean here?

Reply:
util.GetClusterID(npMgr.nodeName), was in next line, why its not in same line?